packages/load/src/load-typedefs.ts

```diff
@@ -135,13 +135,7 @@ function prepareResult({
   const pointerList = Object.keys(pointerOptionMap);
 
   if (pointerList.length > 0 && validSources.length === 0) {
-    throw new Error(`
-      Unable to find any GraphQL type definitions for the following pointers:
```

Comment:
It seems we lose the indentation here?

Reply:
I've left a note about formatting here: https://github.com/ardatan/graphql-tools/pull/7093#discussion_r2033151134
I think it could look better if it doesn't have the indentation & left-aligned. Otherwise, when it'd have some big spaces if an error happens in Codegen:

<img width="844" alt="Screenshot 2025-04-09 at 11 52 38 pm" src="https://github.com/user-attachments/assets/fea35428-8c1b-4b17-a873-d94708d45ec5" />

This is personal preference of course, I'll revert this to preserve the existing behaviour 🙂